### PR TITLE
Cleanup port stuff

### PR DESCRIPTION
**Pull request by benjaoming**
Fix for:

#4608
#4599

Rids this warning during kalite start:

    WARNING:kalite:App with label django_cherrypy_wsgiserver is missing a models.py module.

Removes lots of stale code.

### CODE REVIEW

**Comment by MCGallaspy** [comment]
What's the story here? Need a review?

**Comment by benjaoming** [comment]
This might actually be done and ready, but I had some more ideas while doing it about consolidating the settings. I'll let you know once it's ready, haven't been able to look at it since I left it.

I'm pretty sure that removing the setting of system time (clock set feature) was absolutely necessary and I think we agreed it was a good idea since it didn't actually work anyways and was out of KA Lite's scope of features.

**Comment by benjaoming** [comment]
Now the message from `kalite start` is this:

```
➜  ka-lite git:(cleanup-port-stuff) bin/kalite start --port=7007 --settings=kalite.project.settings.raspberry_pi
Running 'kalite start' as daemon (system service)

Stand by while the server loads its data...

To access KA Lite from another connected computer, try the following address(es):
	http://192.168.40.169:7007/
	http://192.168.40.148:7007/
To access KA Lite from this machine, try the following address:
	http://127.0.0.1:7007/


KA Lite configured behind another server, primary addresses are:

	http://192.168.40.169:8008/
	http://192.168.40.148:8008/
	http://127.0.0.1:8008/
```

**Comment by benjaoming** [comment]
@aronasorman assigning you in case you can second-guess pushing this into 0.16, no stress if not, but then I would recommend creating a 0.16.x branch so we can start merging stuff for a 0.16.1 release

File: kalite/django_cherrypy_wsgiserver/cherrypyserver.py [hunk 1]
```diff
@@ -158,13 +158,13 @@ def port_is_available(host, port):
 
 
 def run_cherrypy_server(host="127.0.0.1", port=None, threads=None, daemonize=False, pidfile=None, autoreload=False, startuplock=None):
-    port = port or getattr(settings, "PRODUCTION_PORT", 8008)
+    port = port
```
Comment (MCGallaspy) [inline kalite/django_cherrypy_wsgiserver/cherrypyserver.py]: This line is now redundant?

File: kalite/settings/__init__.py [hunk 2]
```diff
@@ -46,25 +46,6 @@ def package_selected(package_name):
...
-
-    # nginx proxy will normally be on 8008 and production port on 7007
-    # If ports are overridden in local_settings, run the optimizerpi script
-    PRODUCTION_PORT = getattr(local_settings, "PRODUCTION_PORT", 7007)
-    PROXY_PORT = getattr(local_settings, "PROXY_PORT", 8008)
-    assert PRODUCTION_PORT != PROXY_PORT, "PRODUCTION_PORT and PROXY_PORT must not be the same"
-    #SYNCING_THROTTLE_WAIT_TIME = getattr(local_settings, "SYNCING_THROTTLE_WAIT_TIME", 1.0)
-    #SYNCING_MAX_RECORDS_PER_REQUEST = getattr(local_settings, "SYNCING_MAX_RECORDS_PER_REQUEST", 10)
-
-    PASSWORD_ITERATIONS_TEACHER = getattr(local_settings, "PASSWORD_ITERATIONS_TEACHER", 2000)
```
Comment (MCGallaspy) [inline kalite/settings/__init__.py]: Used anywhere?

**Comment by MCGallaspy** [comment]
:+1:

File: kalite/settings/__init__.py (hunk 2 above)
Comment (benjaoming) [inline kalite/settings/__init__.py]: Nope, I verified it. This whole pattern is so obsolete that all of the "config packages" are better off being deleted now.

File: kalite/django_cherrypy_wsgiserver/cherrypyserver.py (hunk 1 above)
Comment (benjaoming) [inline kalite/django_cherrypy_wsgiserver/cherrypyserver.py]: Actually it's inside a function that's not being called from anywhere. Will clean this up instead.

**Comment by benjaoming** [comment]
That line `port = port` made me realize that we had a whole stale app, `django_cherrypy_wsgiserver` so I went ahead and got rid of it :)

**Comment by benjaoming** [comment]
Rebased on latest develop

**Pull request merged**